extract-wos-article.py

```diff
@@ -23,12 +30,52 @@
     "Current Publisher": "publisher",
 }
 
-YEAR_REGEX = re.compile(".*([0-9]{4})")
+REGEX = {
+    "year": re.compile(".*([0-9]{4})"),
+    "wos_id": re.compile("(WOS:[0-9]{15})"),
+}
+CITING_SUMMARY_URL_PREFIX = "https://www.webofscience.com/wos/woscc/citing-summary/"
+
+
+async def extract_citing_summary(page: Page, article_id: str):
+    url = f"{CITING_SUMMARY_URL_PREFIX}{article_id}"
```

Comment:
```suggestion
    url = f"{CITING_SUMMARY_URL_PREFIX}/{article_id}"
```

@JMatej I'd probably even explicitly leave the last separator in here, so that it's obvious what's happening. Feel free to just ignore it though :slightly_smiling_face:

Reply:
(Note that `CITING_SUMMARY_URL_PREFIX` would need to change as well)